### ISSUE TITLE
Settings view: build SettingsOptionView

### ISSUE THREAD

**Issue by mrlucciola** [mrlucciola/link-hack-ccip#99]
Blocked by #26 

## Description

`SettingsOptionView` is a nested layout component view within `Settings`. When state var `settings.currentView` === `<name-of-option>` the `switch` statement within the root `Settings` component will `return` the clicked component (a `<SettingsOptionLayout>` component - see below).

## Definition of done

- Create component `SettingsOptionView` 
- (Separate ticket) Create component `SettingsOptionLayout`
- (Separate ticket) Create state store for `Settings` with observable `currentView`. This has options `menu` and `option`. At the root `Settings` component,  with values `menu` for the `SettingsMenu` `option`
